Nævner at hver node har en embedding (#39)
thesis20/thesis_paper: sections/csgcn/csgcn-chapter.tex

\subsection{Model architecture}\label{subsec:csgcn_is_model_architecture}
Based on the considerations made in \Cref{sec:foundational_knowledge}, the CSGCN models are based on these observations made in SGCN \cite{SimplifyingGCN} and the LightGCN model \cite{LightGCN} about simplifying the models to a linear model.

> To facilitate the model architecture, each node in the graph is represented by a low-dimensional vector, or a so called embedding.

Reviewer: ```suggestion
To facilitate the model architecture, each node in the graph is represented by a low-dimensional vector, also called an embedding.
```
